src/vs/platform/telemetry/common/1dsAppender.ts

```diff
@@ -18,6 +18,7 @@ export interface IAppInsightsCore {
 }
 
 const endpointUrl = 'https://mobile.events.data.microsoft.com/OneCollector/1.0';
+const endpointHealthUrl = 'https://mobile.events.data.microsoft.com/ping';
```

Comment:
Should this live in product.json?

Reply:
We've always had this in product so I haven't bothered to change it, but I think in the future we might want to move it. It's a public domain that isn't super helpful without the ingestion key and the ingestion key is stored in product.json.